src/ranged.cpp

```diff
@@ -253,6 +253,29 @@ bool player::handle_gun_damage( item &it )
     return true;
 }
 
+void npc::pretend_fire( npc *source, int shots, item &gun )
+{
+    int curshot = 0;
+    if( g->u.sees( *source ) && one_in( 50 ) ) {
+        add_msg( m_info, _( "%1$s shoots something." ), this->disp_name().c_str() );
```

Comment:
```suggestion
        add_msg( m_info, _( "%1$s shoots something." ), this->disp_name() );
```

Reply:
I don't see a reason to include `1$` if there is no other placeholders.